Makefile.gen.app.mk

```diff
@@ -26,16 +26,9 @@ lint-chart: check-env ## Runs ct against the default chart.
 
 update-chart: check-env ## Sync chart with upstream repo.
 	@echo "====> $@"
+	rm -rf ./helm/$(APPLICATION)
```

Comment:
vendir does an `rm` of all the target directory while syncing. I think you could rework your vendir to avoid this change by instead of copying to a temp directory, targeting the dst directory like [this](https://github.com/giantswarm/ingress-nginx-app/blob/main/vendir.yml#L6)
You'll still need the sed though